From: Jiri Slaby <jslaby@suse.cz>
Date: Wed, 24 Jul 2019 16:34:48 +0000 (+0100)
Subject: Fix build after y2038 changes in glib.
X-Git-Tag: v2.81rc1~79
X-Git-Url: http://thekelleys.org.uk/gitweb/?p=dnsmasq.git;a=commitdiff_plain;h=3052ce208acf602f0163166dcefb7330d537cedb;hp=18e17665fdc03711c3759df464b1197015727cc7

Fix build after y2038 changes in glib.

SIOCGSTAMP is defined in linux/sockios.h, not asm/sockios.h now.
---

diff --git a/src/dnsmasq.h b/src/dnsmasq.h
index ff3204a..3ef04ad 100644
--- a/src/dnsmasq.h
+++ b/src/dnsmasq.h
@@ -137,6 +137,7 @@ typedef unsigned long long u64;
 #endif
 
 #if defined(HAVE_LINUX_NETWORK)
+#include <linux/sockios.h>
 #include <linux/capability.h>
 /* There doesn't seem to be a universally-available 
    userspace header for these. */
